.github/workflows/on-push.yml

```diff
@@ -5,8 +5,6 @@ on:
     branches:
       - master
   pull_request:
-    branches:
```

Comment:
intentional deletion?

Reply:
Yup, this was intentional. Essentially if there is a push into master run the tests, and if a pull request is made in any case run the tests.

Before, the tests would only run on PRs if the base branch was master. Which meant this PR wouldn't have its tests run.